internal/monitors/filesystems/filesystems.go

```diff
@@ -77,6 +89,10 @@ func (m *Monitor) getCommonDimensions(partition *gopsutil.PartitionStat) map[str
 		"device":          strings.Replace(partition.Device, " ", "_", -1),
 		"plugin_instance": "",
 	}
+	if m.hostFSPath != "" {
+		// strip configured hostFSPath from mountpoint
+		dims["mountpoint"] = strings.Replace(partition.Mountpoint, m.hostFSPath, "", 1)
```

Comment:
Shouldn't this be `dims["mountpoint"] = strings.Replace(dims["mountpoint"],...)` to take advantage of the space to underscore conversion above?